app/konnect/getting-started/configure-runtime.md

```diff
@@ -15,16 +15,15 @@ runtime doesn't need a database to store configuration data. Instead,
 configuration is stored in-memory on each node, and you can easily update
 multiple runtimes from one {{site.konnect_short_name}} account with a few clicks.
 
-<div class="alert alert-ee blue">
-<b>Note:</b> Kong does not host runtimes. You must install and host your own
+{:.note}
+> **Note:** Kong does not host runtimes. You must install and host your own
 runtime instances.
-</div>
 
 ## Prerequisites
 
 * You have **Runtime Admin** or **Organization Admin** permissions in
 {{site.konnect_saas}}. If you created this account, the account has
-organization admin permissions by default.
+organization admin permissions by default. _[To do: check what the new permissions are]_
```

Comment:
Yes, this needs to be re-worded once we start looking into the teams and roles aspect of Helium.
The short summary here are that:
- The user must have Runtime Group Admin or Deployer role. (This part may not need to be explained in the getting started guide, but I wanted to point out the differenced between Role and Teams)
- This role is already assigned to these pre-defined teams: Organization Admin and Runtime Admin.
- The creator of the org will by default be in the Organization Admin team.